web/packages/teleport/src/services/joinToken/joinToken.ts

```diff
@@ -42,15 +44,17 @@ class JoinTokenService {
             req.suggestedAgentMatcherLabels
           ),
         },
-        signal
+        signal,
+        mfaResponse
       )
       .then(makeJoinToken);
```

Comment:
Can we not mix async/await and `.then()` in one function? It makes it harder to reason about the flow. I'd be grateful if you converted these calls.